From 1617ab00508e3722ed55d3ec48468e19a95b17b9 Mon Sep 17 00:00:00 2001
From: Jianzheng Zhou <jianzheng.zhou@freescale.com>
Date: Mon, 9 Dec 2013 14:05:30 +0800
Subject: [PATCH 31/32] ENGR00290619 Ethernet:Tune ethernet performance

fix ethernet and wifi can't be tuned seperately issue.
This commit is already merged by Google. So can be ignored in next update.
https://android-review.googlesource.com/#/c/70130/

Signed-off-by: Jianzheng Zhou <jianzheng.zhou@freescale.com>
---
 core/java/android/net/EthernetDataTracker.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/java/android/net/EthernetDataTracker.java b/core/java/android/net/EthernetDataTracker.java
index 6988b5e..5c88546 100644
--- a/core/java/android/net/EthernetDataTracker.java
+++ b/core/java/android/net/EthernetDataTracker.java
@@ -421,7 +421,7 @@ public class EthernetDataTracker extends BaseNetworkStateTracker {
      * for this network.
      */
     public String getTcpBufferSizesPropName() {
-        return "net.tcp.buffersize.wifi";
+        return "net.tcp.buffersize.ethernet";
     }
 
     public void setDependencyMet(boolean met) {
-- 
1.8.0

